front/app/components/SeatInfo/SeatInfo.test.tsx

```diff
@@ -65,12 +70,23 @@ describe('SeatInfo', () => {
     expect(screen.queryByText('Additional seats')).not.toBeInTheDocument();
   });
 
-  it('shows additional seats when user has used more', () => {
+  it('shows collaborators additional seats when user has used more', () => {
     mockUserSeatsData.data.attributes.project_moderators_number = 15;
+    mockAppConfiguration.data.attributes.settings.core.additional_moderators_number = 7;
     render(<SeatInfo seatType="collaborator" />);
     expect(screen.getByText('Current collaborator seats')).toBeInTheDocument();
     expect(screen.getByText('9/9')).toBeInTheDocument();
-    expect(screen.getByText('6')).toBeInTheDocument();
+    expect(screen.getByText('6/7')).toBeInTheDocument();
+    expect(screen.queryByText('Additional seats')).toBeInTheDocument();
+  });
+
```

Comment:
Maybe it's clearer if you separate these with a comments?

```
    // Current seats
    expect(screen.getByText('Current collaborator seats')).toBeInTheDocument();
    expect(screen.getByText('9/9')).toBeInTheDocument();
    
    // Additional seats
    expect(screen.queryByText('Additional seats')).toBeInTheDocument();
    expect(screen.getByText('6/7')).toBeInTheDocument();
```

Reply:
Could you also add a comment how you arrive at the 6/7? It took me a while to grasp.